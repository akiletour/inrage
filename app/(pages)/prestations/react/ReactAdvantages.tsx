import NextJsAdvantageCard from '@component/items/NextJsAdvantageCard'

export default function ReactAdvantages() {
  const advantages = [
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2L2 7v10c0 5.55 3.84 9.739 9 11 5.16-1.261 9-5.45 9-11V7l-10-5z" />
        </svg>
      ),
      title: 'Performance Optimisée',
      description:
        'Next.js offre des optimisations automatiques avec le SSR, SSG et le code splitting pour des temps de chargement ultra-rapides.',
      highlight: 'Core Web Vitals',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-2 15l-5-5 1.41-1.41L10 14.17l7.59-7.59L19 8l-9 9z" />
        </svg>
      ),
      title: 'SEO Natif',
      description:
        'Le rendu côté serveur et la génération statique garantissent une indexation parfaite par les moteurs de recherche.',
      highlight: 'Google Ready',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M19.35 10.04C18.67 6.59 15.64 4 12 4 9.11 4 6.6 5.64 5.35 8.04 2.34 8.36 0 10.91 0 14c0 3.31 2.69 6 6 6h13c2.76 0 5-2.24 5-5 0-2.64-2.05-4.78-4.65-4.96z" />
        </svg>
      ),
      title: 'Scalabilité',
      description:
        'Architecture composants modulaire permettant de faire évoluer facilement vos applications avec votre business.',
      highlight: 'Enterprise Ready',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2C13.1 2 14 2.9 14 4C14 5.1 13.1 6 12 6C10.9 6 10 5.1 10 4C10 2.9 10.9 2 12 2ZM21 9V7L15 1L13.5 2.5L16.17 5.24C15.24 5.9 15 6.32 15 7V9H21ZM3 23H5L14 13.91V22H8V19H6V22H0V19C0 18.46 0.22 17.94 0.61 17.55L7.05 11.1L7.55 12.1H9V14C9 14.55 8.78 15.05 8.39 15.44L6 17.83V19H8V16.5L9.5 15H11.24L3 23Z" />
        </svg>
      ),
      title: 'Développement Moderne',
      description:
        'TypeScript intégré, Hot Module Replacement et outils de développement avancés pour une productivité maximale.',
      highlight: 'Developer Experience',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M12 2l3.09 6.26L22 9.27l-5 4.87 1.18 6.88L12 17.77l-6.18 3.25L7 14.14 2 9.27l6.91-1.01L12 2z" />
        </svg>
      ),
      title: 'Écosystème Riche',
      description:
        'Vaste communauté, librairies maintenues et intégrations tierces pour accélérer le développement.',
      highlight: 'Open Source',
    },
    {
      icon: (
        <svg
          className="w-16 h-16 text-white"
          fill="currentColor"
          viewBox="0 0 24 24"
        >
          <path d="M20 4H4C2.9 4 2 4.9 2 6v12c0 1.1.9 2 2 2h16c1.1 0 2-.9 2-2V6c0-1.1-.9-2-2-2zm0 14H4V8h16v10z" />
        </svg>
      ),
      title: 'Responsive Design',
      description:
        "Composants adaptatifs par défaut et optimisations mobiles intégrées pour toutes les tailles d'écran.",
      highlight: 'Mobile First',
    },
  ]

  return (
    <div className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 gap-6 mt-16">
      {advantages.map((advantage, index) => (
        <div
          key={index}
          className="animate-in slide-in-from-bottom-4 duration-700"
          style={{ animationDelay: `${index * 150}ms` }}
        >
          <NextJsAdvantageCard
            icon={advantage.icon}
            title={advantage.title}
            description={advantage.description}
            highlight={advantage.highlight}
          />
        </div>
      ))}
    </div>
  )
}
