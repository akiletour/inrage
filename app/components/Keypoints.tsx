import Image from 'next/image'

import Diagonal from '@component/Diagonal'
import KeypointItem from '@component/items/KeypointItem'

import AchieveIcon from './icons/AchieveIcon'
import CupIcon from './icons/CupIcon'
import LeafHeartIcon from './icons/LeafHeartIcon'
import ThumbIcon from './icons/ThumbIcon'

export default function Keypoints() {
  return (
    <div className="relative">
      <Diagonal
        className="h-20 sm:h-40 lg:h-auto"
        flipX
        flipY
        bgClass="fill-gray-dark"
        bgCorner="fill-orange"
      />
      <Image
        className="-z-10"
        src="/images/yohann-tilotti_bloody-sky.jpeg"
        alt="Yohann Tilotti - Bloody Sky"
        fill
        sizes="100vw"
      />
      <div className="container grid grid-cols-2 md:grid-cols-4 gap-y-8 py-8">
        <KeypointItem
          icon={<CupIcon className="h-6 md:h-10" />}
          value="2356"
          label="tasses à café"
        />
        <KeypointItem
          icon={<ThumbIcon className="h-6 md:h-10" />}
          value="352"
          label="projets"
        />
        <KeypointItem
          icon={<AchieveIcon className="h-6 md:h-10" />}
          value="15 années"
          label="d'expérience"
        />
        <KeypointItem
          icon={<LeafHeartIcon className="h-6 md:h-10" />}
          value="150+"
          label="idées à développer"
        />
      </div>
      <Diagonal
        className="h-20 sm:h-40 lg:h-auto"
        bgClass="fill-gray-darker"
        bgCorner="fill-orange"
      />
    </div>
  )
}
