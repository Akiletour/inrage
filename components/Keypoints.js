import Image from 'next/image';
import Diagonal from './layouts/Diagonal';
import KeypointItem from './items/KeypointItem';

export default function Keypoints() {
  return (
    <div className="relative">
      <Diagonal
        className="h-10 sm:h-20 lg:h-auto"
        flipX
        flipY
        bgClass="fill-gray-dark"
        bgCorner="fill-orange"
      />
      <Image className="-z-10" src="/images/yohann-tilotti_bloody-sky.jpeg" layout="fill" alt="Yohann Tilotti - Bloody Sky" />
      <div className="container grid grid-cols-2 md:grid-cols-4 gap-y-4 py-4">
        <KeypointItem />
        <KeypointItem />
        <KeypointItem />
        <KeypointItem />
      </div>
      <Diagonal
        className="h-10 sm:h-20 lg:h-auto"
        bgClass="fill-gray-darker"
        bgCorner="fill-orange"
      />
    </div>
  );
}
