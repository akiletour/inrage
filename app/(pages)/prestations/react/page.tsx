import Image from 'next/image'

import ContactForm from '@component/ContactForm'
import Diagonal from '@component/Diagonal'
import Layout from '@component/Layout'
import SectionTitle from '@component/SectionTitle'
import ReactServicesGrid from './ReactServicesGrid'
import ReactAdvantages from './ReactAdvantages'
import ReactProjectsShowcase from './ReactProjectsShowcase'
import { RouteLink, getCanonicalUrl } from '@lib/router'

export const metadata = {
  title: 'Développeur React Freelance La Rochelle | Expert Next.js - inRage',
  description:
    "Développeur React freelance expert à La Rochelle. Création d'applications Next.js modernes et performantes. Spécialisé TypeScript, SSR, e-commerce. Devis gratuit.",
  keywords: [
    'développeur react freelance la rochelle',
    'expert nextjs la rochelle',
    'création application react',
    'développement spa react',
    'freelance typescript la rochelle',
    'développeur frontend react france',
  ],
  alternates: {
    canonical: getCanonicalUrl(RouteLink.prestationReact),
  },
  openGraph: {
    title: 'Développeur React Freelance La Rochelle | Expert Next.js',
    description:
      "Développeur React freelance expert à La Rochelle. Création d'applications Next.js modernes et performantes pour votre business.",
    url: getCanonicalUrl(RouteLink.prestationReact),
    siteName: 'inRage',
    locale: 'fr_FR',
    type: 'website',
    images: [
      {
        url: 'https://www.inrage.fr/images/react-nextjs-development.png',
        width: 697,
        height: 736,
        alt: 'Développement React Next.js applications modernes La Rochelle',
      },
    ],
  },
  twitter: {
    card: 'summary_large_image',
    title: 'Développeur React Freelance La Rochelle | Expert Next.js',
    description:
      "Développeur React freelance expert à La Rochelle. Création d'applications Next.js modernes et performantes.",
    images: ['https://www.inrage.fr/images/react-nextjs-development.png'],
  },
  robots: {
    index: true,
    follow: true,
    googleBot: {
      index: true,
      follow: true,
      'max-video-preview': -1,
      'max-image-preview': 'large',
      'max-snippet': -1,
    },
  },
}

export default function PrestationReact() {
  const reactServiceSchema = {
    '@context': 'https://schema.org',
    '@type': 'Service',
    name: 'Développement React et Next.js',
    description:
      "Service de développement d'applications web modernes avec React, Next.js et TypeScript par un développeur freelance expert basé à La Rochelle",
    provider: {
      '@type': 'Person',
      name: 'Pascal Gault',
      jobTitle: 'Développeur React Freelance',
      address: {
        '@type': 'PostalAddress',
        addressLocality: 'La Rochelle',
        addressRegion: 'Nouvelle-Aquitaine',
        postalCode: '17000',
        addressCountry: 'FR',
      },
      url: 'https://www.inrage.fr',
      sameAs: [
        'https://www.linkedin.com/in/akiletour/',
        'https://github.com/akiletour/inrage',
      ],
    },
    serviceType: 'Développement Web',
    category: "Développement d'applications",
    offers: {
      '@type': 'Offer',
      description: "Développement d'applications React et Next.js",
      price: 'Sur devis',
      priceCurrency: 'EUR',
      availability: 'https://schema.org/InStock',
    },
    areaServed: {
      '@type': 'Place',
      name: 'France',
      containedInPlace: {
        '@type': 'Place',
        name: 'La Rochelle',
      },
    },
    hasOfferCatalog: {
      '@type': 'OfferCatalog',
      name: 'Services React & Next.js',
      itemListElement: [
        {
          '@type': 'Offer',
          itemOffered: {
            '@type': 'Service',
            name: "Création d'applications React",
            description:
              "Développement d'applications single-page modernes avec React",
          },
        },
        {
          '@type': 'Offer',
          itemOffered: {
            '@type': 'Service',
            name: 'Développement Next.js',
            description:
              'Applications Next.js avec SSR/SSG pour performances optimales',
          },
        },
        {
          '@type': 'Offer',
          itemOffered: {
            '@type': 'Service',
            name: 'Migration vers React',
            description:
              "Migration d'applications existantes vers l'écosystème React",
          },
        },
      ],
    },
  }

  const breadcrumbSchema = {
    '@context': 'https://schema.org',
    '@type': 'BreadcrumbList',
    itemListElement: [
      {
        '@type': 'ListItem',
        position: 1,
        name: 'Accueil',
        item: 'https://www.inrage.fr',
      },
      {
        '@type': 'ListItem',
        position: 2,
        name: 'Prestations',
        item: 'https://www.inrage.fr/prestations',
      },
      {
        '@type': 'ListItem',
        position: 3,
        name: 'Développement React & Next.js',
        item: 'https://www.inrage.fr/prestations/react',
      },
    ],
  }

  const faqSchema = {
    '@context': 'https://schema.org',
    '@type': 'FAQPage',
    mainEntity: [
      {
        '@type': 'Question',
        name: 'Pourquoi choisir un développeur React freelance à La Rochelle ?',
        acceptedAnswer: {
          '@type': 'Answer',
          text: 'Un développeur React freelance à La Rochelle offre une expertise locale avec une connaissance approfondie du marché français. Je propose un accompagnement personnalisé, des délais flexibles et une communication directe pour vos projets React et Next.js.',
        },
      },
      {
        '@type': 'Question',
        name: 'Quels sont les avantages de React et Next.js pour mon projet ?',
        acceptedAnswer: {
          '@type': 'Answer',
          text: 'React et Next.js offrent des performances excellentes, un SEO optimisé avec le rendu côté serveur, une évolutivité maximale et une expérience utilisateur fluide. Ces technologies modernes garantissent la pérennité de votre application web.',
        },
      },
      {
        '@type': 'Question',
        name: "Combien coûte le développement d'une application React ?",
        acceptedAnswer: {
          '@type': 'Answer',
          text: "Le coût dépend de la complexité du projet. Une application React simple commence à partir de 3000€, tandis qu'une solution complète avec Next.js peut varier de 5000€ à 15000€. Je propose toujours un devis gratuit personnalisé.",
        },
      },
      {
        '@type': 'Question',
        name: "Quels types d'applications React développez-vous ?",
        acceptedAnswer: {
          '@type': 'Answer',
          text: "Je développe tous types d'applications : SPA (Single Page Applications), sites e-commerce, dashboards d'administration, plateformes SaaS, applications métier et sites vitrines modernes avec React et Next.js.",
        },
      },
    ],
  }

  return (
    <>
      <script
        type="application/ld+json"
        dangerouslySetInnerHTML={{ __html: JSON.stringify(reactServiceSchema) }}
      />
      <script
        type="application/ld+json"
        dangerouslySetInnerHTML={{ __html: JSON.stringify(breadcrumbSchema) }}
      />
      <script
        type="application/ld+json"
        dangerouslySetInnerHTML={{ __html: JSON.stringify(faqSchema) }}
      />
      <Layout
        breadcrumbs={[
          {
            link: RouteLink.prestations,
            title: 'Prestations sous WordPress, Prestashop et Symfony',
          },
        ]}
        title="Développement React & Next.js"
      >
        <div className="flex max-lg:flex-col relative container gap-16 *:flex-1">
          <Image
            className="h-full! object-cover drop-shadow-2xl"
            src="/images/react-nextjs-development.webp"
            width={697}
            priority
            height={736}
            alt="Développeur React freelance La Rochelle - Création applications Next.js modernes"
            style={{
              maxWidth: '100%',
              height: 'auto',
            }}
          />
          <div className=" relative flex items-center">
            <div className="space-y-12">
              <div className="space-y-8">
                <h1 className="font-bold text-white text-2xl md:text-4xl lg:text-5xl leading-tight animate-in slide-in-from-bottom-4 duration-700">
                  Développeur React freelance à
                  <span className="text-orange"> La Rochelle</span>
                </h1>

                <div className="text-gray-light font-medium text-lg md:text-xl leading-relaxed animate-in slide-in-from-bottom-6 duration-700 delay-150">
                  <p>
                    Expert React et Next.js freelance basé à La Rochelle,
                    spécialisé dans la création d&apos;applications web
                    performantes avec React, Next.js et TypeScript.
                  </p>
                </div>
              </div>

              <div className="space-y-8 text-gray-light/90 leading-relaxed animate-in slide-in-from-bottom-8 duration-700 delay-300">
                <p>
                  Développeur React freelance à La Rochelle, je conçois et
                  développe des applications web modernes, réactives et
                  scalables en utilisant l&apos;écosystème React. De
                  l&apos;application single-page au site e-commerce complexe, en
                  passant par les dashboards et les plateformes SaaS pour
                  entreprises françaises.
                </p>
                <p>
                  Mon expertise en tant que développeur React freelance La
                  Rochelle couvre React, Next.js, TypeScript, ainsi que
                  l&apos;intégration d&apos;APIs REST et GraphQL pour créer des
                  expériences utilisateur fluides et performantes adaptées au
                  marché français.
                </p>
                <p>
                  Je vous accompagne dans la modernisation de vos outils
                  existants ou la création de nouvelles applications avec les
                  technologies les plus récentes de l&apos;écosystème React.
                </p>
              </div>

              <div className="flex flex-col sm:flex-row gap-8 pt-8 animate-in slide-in-from-bottom-10 duration-700 delay-500">
                <a
                  href="#contact"
                  className="button-primary group inline-flex items-center justify-center px-12 py-6 rounded-lg font-semibold text-white bg-orange hover:bg-orange-dark transform hover:scale-105 transition-all duration-300 shadow-lg hover:shadow-orange/25"
                >
                  Démarrer mon projet
                  <svg
                    className="w-10 h-10 ml-4 transform group-hover:translate-x-2 transition-transform"
                    fill="none"
                    stroke="currentColor"
                    viewBox="0 0 24 24"
                  >
                    <path
                      strokeLinecap="round"
                      strokeLinejoin="round"
                      strokeWidth={2}
                      d="M17 8l4 4m0 0l-4 4m4-4H3"
                    />
                  </svg>
                </a>
                <a
                  href="#services"
                  className="button-secondary inline-flex items-center justify-center px-12 py-6 rounded-lg font-semibold text-orange border-2 border-orange hover:bg-orange hover:text-white transition-all duration-300"
                >
                  Voir mes services
                </a>
              </div>
            </div>
          </div>
        </div>

        <div className="bg-gray-darker mt-20 relative overflow-hidden">
          <div className="absolute inset-0 bg-gray-darker/80"></div>
          <Diagonal
            flipX
            flipY
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgClass="fill-gray-dark"
            bgCorner="fill-gray-darker"
          />
          <div className="container text-center relative z-10">
            <div className="md:px-36 text-lg max-w-4xl mx-auto">
              <h2 className="text-white text-3xl md:text-4xl lg:text-5xl font-bold mb-16 leading-tight">
                Développement React à La Rochelle
                <br />
                <span className="text-orange">au service de vos projets</span>
              </h2>

              <div className="grid grid-cols-1 md:grid-cols-3 gap-16 mt-24">
                <div className="text-center group">
                  <div className="w-32 h-32 mx-auto mb-8 bg-orange rounded-2xl flex items-center justify-center transform group-hover:scale-110 transition-transform duration-300">
                    <svg
                      className="w-16 h-16 text-white"
                      fill="currentColor"
                      viewBox="0 0 24 24"
                    >
                      <path d="M9.4 16.6L4.8 12l4.6-4.6L8 6l-6 6 6 6 1.4-1.4zm5.2 0L19.2 12l-4.6-4.6L16 6l6 6-6 6-1.4-1.4zM12 18l2-8h-4l2 8z" />
                    </svg>
                  </div>
                  <h3 className="text-white font-semibold text-lg mb-4">
                    Technologies modernes
                  </h3>
                  <p className="text-gray-light text-sm leading-relaxed">
                    React et Next.js représentent aujourd&apos;hui les
                    technologies de référence pour le développement
                    d&apos;applications web modernes.
                  </p>
                </div>

                <div className="text-center group">
                  <div className="w-32 h-32 mx-auto mb-8 bg-orange rounded-2xl flex items-center justify-center transform group-hover:scale-110 transition-transform duration-300">
                    <svg
                      className="w-16 h-16 text-white"
                      fill="currentColor"
                      viewBox="0 0 24 24"
                    >
                      <path d="M12 2l3.5 6.5h7L17 14l2.5 7.5L12 18l-7.5 3.5L7 14 1.5 8.5h7L12 2zm0 2.8L9.5 9H4.2l3.3 3.7L6 19l6-3.2L18 19l-1.5-6.3L20.8 9h-5.3L12 4.8z" />
                    </svg>
                  </div>
                  <h3 className="text-white font-semibold text-lg mb-4">
                    Approche component-driven
                  </h3>
                  <p className="text-gray-light text-sm leading-relaxed">
                    J&apos;utilise une approche component-driven pour créer des
                    interfaces réutilisables et maintenables avec une
                    architecture solide.
                  </p>
                </div>

                <div className="text-center group">
                  <div className="w-32 h-32 mx-auto mb-8 bg-orange rounded-2xl flex items-center justify-center transform group-hover:scale-110 transition-transform duration-300">
                    <svg
                      className="w-16 h-16 text-white"
                      fill="currentColor"
                      viewBox="0 0 24 24"
                    >
                      <path d="M12 2C6.48 2 2 6.48 2 12s4.48 10 10 10 10-4.48 10-10S17.52 2 12 2zm-1 17.93c-3.94-.49-7-3.85-7-7.93 0-.62.08-1.21.21-1.79L9 15v1c0 1.1.9 2 2 2v1.93zm6.9-2.54c-.26-.81-1-1.39-1.9-1.39h-1v-3c0-.55-.45-1-1-1H8v-2h2c.55 0 1-.45 1-1V7h2c1.1 0 2-.9 2-2v-.41c2.93 1.19 5 4.06 5 7.41 0 2.08-.8 3.97-2.1 5.39z" />
                    </svg>
                  </div>
                  <h3 className="text-white font-semibold text-lg mb-4">
                    Adaptabilité
                  </h3>
                  <p className="text-gray-light text-sm leading-relaxed">
                    Que ce soit pour un site vitrine, une application e-commerce
                    ou un dashboard, je m&apos;adapte à vos besoins spécifiques.
                  </p>
                </div>
              </div>
            </div>
          </div>
          <Diagonal
            bgClass="fill-gray-dark"
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgCorner="fill-gray-darker"
          />
        </div>

        <div
          id="services"
          className="container mt-8 md:mt-0 lg:-mb-20 relative z-10 scroll-mt-40"
        >
          <div className="animate-in slide-in-from-bottom-4 duration-700">
            <SectionTitle
              title={['Mes services', 'React & Next.js']}
              content="De la conception à la mise en production, je vous accompagne dans la création d'applications React performantes et modernes adaptées à vos besoins business."
            />
          </div>

          <ReactServicesGrid />
        </div>

        <Diagonal
          className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
          bgCorner="fill-orange"
          bgClass="fill-gray-darker"
        />

        <div className="bg-gray-darker relative overflow-hidden">
          <div className="absolute inset-0 bg-gray-darker/80"></div>
          <div className="container mb-8 md:mb-0 relative z-10">
            <div className="animate-in slide-in-from-bottom-4 duration-700">
              <SectionTitle
                content="Découvrez pourquoi React et Next.js sont les technologies de choix pour vos projets web modernes."
                title="Avantages React & Next.js"
              />
            </div>

            <ReactAdvantages />
          </div>

          <Diagonal
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgCorner="fill-orange"
            bgClass="fill-gray-dark"
          />
        </div>

        <div className="container mt-8 md:mt-0">
          <div className="animate-in slide-in-from-bottom-4 duration-700">
            <SectionTitle
              content="Découvrez quelques exemples de réalisations React et Next.js que j'ai développées pour mes clients."
              title="Projets React & Next.js"
            />
          </div>

          <ReactProjectsShowcase />
        </div>

        <Diagonal
          className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
          bgCorner="fill-orange"
          bgClass="fill-gray-darker"
        />

        <div
          id="contact"
          className="bg-gray-darker relative overflow-hidden scroll-mt-40"
        >
          <div className="absolute inset-0 bg-gray-darker/80"></div>
          <div className="container relative z-10">
            <div className="animate-in slide-in-from-bottom-4 duration-700">
              <SectionTitle
                content="Prêt à moderniser vos applications web avec React et Next.js ? Contactez-moi pour discuter de votre projet."
                title="Démarrons votre projet"
              />
            </div>

            <div className="animate-in slide-in-from-bottom-6 duration-700 delay-200">
              <ContactForm />
            </div>
          </div>

          <Diagonal
            className="h-20 sm:h-32 md:h-50 lg:h-72 xl:h-90 -z-10"
            bgCorner="fill-orange"
            bgClass="fill-gray-dark"
          />
        </div>
      </Layout>
    </>
  )
}
