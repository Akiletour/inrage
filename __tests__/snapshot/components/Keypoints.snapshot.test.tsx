import { render } from '@testing-library/react';

import Keypoints from '@component/Keypoints';

it('renders Keypoints unchanged', () => {
  const { container } = render(<Keypoints />);
  expect(container).toMatchSnapshot();
});
