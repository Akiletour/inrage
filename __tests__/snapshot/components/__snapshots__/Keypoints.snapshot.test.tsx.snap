// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Keypoints unchanged 1`] = `
<div>
  <div
    class="relative"
  >
    <div
      class="relative"
    >
      <svg
        class="h-10 sm:h-20 lg:h-auto w-full block -scale-x-100 -scale-y-100"
        height="361"
        preserveAspectRatio="xMidYMax slice"
        viewBox="0 0 1920 361"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          class="fill-gray-dark"
          d="M1349 234L1920 0v361H0z"
          fill="#000"
        />
        <path
          class="fill-orange"
          d="M1445 197.5L1920 4v249z"
          fill="#BE4C4C"
        />
      </svg>
    </div>
    <div
      class="container grid grid-cols-2 md:grid-cols-4 gap-y-4 py-4"
    >
      <div
        class="text-center text-white flex items-center flex-col"
      >
        <div
          class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
        >
          <svg
            class="h-3 md:h-5"
            height="24"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M127.1 146.5c1.3 7.7 8 13.5 16 13.5h16.5c9.8 0 17.6-8.5 16.3-18-3.8-28.2-16.4-54.2-36.6-74.7-14.4-14.7-23.6-33.3-26.4-53.5C111.8 5.9 105 0 96.8 0H80.4C70.6 0 63 8.5 64.1 18c3.9 31.9 18 61.3 40.6 84.4 12 12.2 19.7 27.5 22.4 44.1zm112 0c1.3 7.7 8 13.5 16 13.5h16.5c9.8 0 17.6-8.5 16.3-18-3.8-28.2-16.4-54.2-36.6-74.7-14.4-14.7-23.6-33.3-26.4-53.5C223.8 5.9 217 0 208.8 0h-16.4c-9.8 0-17.5 8.5-16.3 18 3.9 31.9 18 61.3 40.6 84.4 12 12.2 19.7 27.5 22.4 44.1zM400 192H32c-17.7 0-32 14.3-32 32v192c0 53 43 96 96 96h192c53 0 96-43 96-96h16c61.8 0 112-50.2 112-112s-50.2-112-112-112zm0 160h-16v-96h16c26.5 0 48 21.5 48 48s-21.5 48-48 48z"
              fill="currentColor"
            />
          </svg>
        </div>
        <div
          class="font-medium text-2xl mt-2"
        >
          2356
        </div>
        <div
          class="leading-5"
        >
          tasses à café
        </div>
      </div>
      <div
        class="text-center text-white flex items-center flex-col"
      >
        <div
          class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
        >
          <svg
            class="h-3 md:h-5"
            height="25"
            viewBox="0 0 512 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M104 224H24c-13.255 0-24 10.745-24 24v240c0 13.255 10.745 24 24 24h80c13.255 0 24-10.745 24-24V248c0-13.255-10.745-24-24-24zM64 472c-13.255 0-24-10.745-24-24s10.745-24 24-24 24 10.745 24 24-10.745 24-24 24zM384 81.452c0 42.416-25.97 66.208-33.277 94.548h101.723c33.397 0 59.397 27.746 59.553 58.098.084 17.938-7.546 37.249-19.439 49.197l-.11.11c9.836 23.337 8.237 56.037-9.308 79.469 8.681 25.895-.069 57.704-16.382 74.757 4.298 17.598 2.244 32.575-6.148 44.632C440.202 511.587 389.616 512 346.839 512l-2.845-.001c-48.287-.017-87.806-17.598-119.56-31.725-15.957-7.099-36.821-15.887-52.651-16.178-6.54-.12-11.783-5.457-11.783-11.998v-213.77c0-3.2 1.282-6.271 3.558-8.521 39.614-39.144 56.648-80.587 89.117-113.111 14.804-14.832 20.188-37.236 25.393-58.902C282.515 39.293 291.817 0 312 0c24 0 72 8 72 81.452z"
              fill="currentColor"
            />
          </svg>
        </div>
        <div
          class="font-medium text-2xl mt-2"
        >
          352
        </div>
        <div
          class="leading-5"
        >
          projets
        </div>
      </div>
      <div
        class="text-center text-white flex items-center flex-col"
      >
        <div
          class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
        >
          <svg
            class="h-3 md:h-5"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M552 64H448V24c0-13.3-10.7-24-24-24H152c-13.3 0-24 10.7-24 24v40H24C10.7 64 0 74.7 0 88v56c0 66.5 77.9 131.7 171.9 142.4C203.3 338.5 240 360 240 360v72h-48c-35.3 0-64 20.7-64 56v12c0 6.6 5.4 12 12 12h296c6.6 0 12-5.4 12-12v-12c0-35.3-28.7-56-64-56h-48v-72s36.7-21.5 68.1-73.6C498.4 275.6 576 210.3 576 144V88c0-13.3-10.7-24-24-24zM64 144v-16h64.2c1 32.6 5.8 61.2 12.8 86.2-47.5-16.4-77-49.9-77-70.2zm448 0c0 20.2-29.4 53.8-77 70.2 7-25 11.8-53.6 12.8-86.2H512v16zm-127.3 4.7-39.6 38.6 9.4 54.6c1.7 9.8-8.7 17.2-17.4 12.6l-49-25.8-49 25.8c-8.8 4.6-19.1-2.9-17.4-12.6l9.4-54.6-39.6-38.6c-7.1-6.9-3.2-19 6.7-20.5l54.8-8 24.5-49.6c4.4-8.9 17.1-8.9 21.5 0l24.5 49.6 54.8 8c9.6 1.5 13.5 13.6 6.4 20.5z"
              fill="currentColor"
            />
          </svg>
        </div>
        <div
          class="font-medium text-2xl mt-2"
        >
          15 années
        </div>
        <div
          class="leading-5"
        >
          d'expérience
        </div>
      </div>
      <div
        class="text-center text-white flex items-center flex-col"
      >
        <div
          class="w-10 md:w-15 h-10 md:h-15 border-2 border-white rounded-full flex items-center justify-center"
        >
          <svg
            class="h-3 md:h-5"
            height="24"
            viewBox="0 0 576 512"
            xmlns="http://www.w3.org/2000/svg"
          >
            <path
              d="M546.2 9.7c-5.6-12.5-21.6-13-28.3-1.2C486.9 62.4 431.4 96 368 96h-80C182 96 96 182 96 288c0 19.2 2.9 37.7 8.1 55.1C31.6 399.5 4.3 463.2 2.4 468c-6.6 16.3 1.2 34.9 17.5 41.6 16.4 6.8 35-1.1 41.8-17.3 1.5-3.6 20.9-47.9 71.9-90.6 32.4 43.9 94 85.8 174.9 77.2C465.5 467.5 576 326.7 576 154.3c0-50.2-10.8-102.2-29.8-144.6zM430.6 294.4 345.9 380c-5.5 5.5-14.3 5.5-19.8 0l-84.7-85.6c-24.6-24.9-23.2-66.1 4.3-89.1 24-20 59.6-16.4 81.6 5.8l8.6 8.7 8.6-8.7c22-22.2 57.7-25.8 81.6-5.8 27.6 22.9 29.1 64.2 4.5 89.1z"
              fill="currentColor"
            />
          </svg>
        </div>
        <div
          class="font-medium text-2xl mt-2"
        >
          150+
        </div>
        <div
          class="leading-5"
        >
          idées à développer
        </div>
      </div>
    </div>
    <div
      class="relative"
    >
      <svg
        class="h-10 sm:h-20 lg:h-auto w-full block"
        height="361"
        preserveAspectRatio="xMidYMax slice"
        viewBox="0 0 1920 361"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          class="fill-gray-darker"
          d="M1349 234L1920 0v361H0z"
          fill="#000"
        />
        <path
          class="fill-orange"
          d="M1445 197.5L1920 4v249z"
          fill="#BE4C4C"
        />
      </svg>
    </div>
  </div>
</div>
`;
